featurebyte/service/observation_table.py

```diff
@@ -232,6 +241,109 @@ async def get_observation_table_upload_task_payload(
             observation_set_storage_path=observation_set_storage_path,
         )
 
+    @staticmethod
+    def get_minimum_iet_sql_expr(
+        entity_column_names: List[str], table_details: TableDetails, source_type: SourceType
+    ) -> Expression:
+        """
+        Get the SQL expression to compute the minimum interval in seconds for each entity.
+
+        Parameters
+        ----------
+        entity_column_names: List[str]
+            List of entity column names
+        table_details: TableDetails
+            Table details of the materialized table
+        source_type: SourceType
+            Source type
+
+        Returns
+        -------
+        str
+        """
+        adapter = get_sql_adapter(source_type)
+        point_in_time_quoted = quoted_identifier(SpecialColumnName.POINT_IN_TIME)
+        previous_point_in_time_quoted = quoted_identifier("PREVIOUS_POINT_IN_TIME")
+
+        window_expression = expressions.Window(
+            this=expressions.Anonymous(this="LAG", expressions=[point_in_time_quoted]),
+            partition_by=[make_literal_value(col_name) for col_name in entity_column_names],
```

Comment:
I think this should be using `quoted_identifier` instead of `make_literal_value` since it's referring to columns.